### ISSUE TITLE
Nests

### ISSUE THREAD

**Issue by Paspartout** [Paspartout/UntitledDuckMod#26]
Adding a nest similar to the one from environmental seems to be a popular request.

Not sure if I really want to implement it though since it may make farming too easy.
Also making ducks and geese able to breed automatically could introduce lag in longer running worlds.

Suggested in #23 and #7

**Comment by 1235788998753568hfjfu** [comment]
Animania Farm's Nests is a good reference. Waterfowl can lay three eggs in the nest. You can adjust the frequency and number of eggs laid and reduce them.Hope you can accept it, thank you very much

**Comment by 1235788998753568hfjfu** [comment]
Hatching is a long wait (the hatching of ducks is about 28 days, and the hatching of geese is about 30-33 days) You can add them to the game. Make the game more interesting.

**Comment by Paspartout** [comment]
Yeah I think the quark feeding trough does check if there are already too much entites around it and won't breed and just consume the food. I think I probably could do something similar to avoid crowded nests. I also like the idea of building bedding like you described but that seems a bit more complicated than just checking if the area is too crowded.

**Comment by Raavea** [comment]
I don't know if the mod I use is the same as Quark's one! It is by someone called Slexom; I don't think Quark is available for Fabric.

I know absolutely nothing about the internals of minecraft mods so maybe this is an untenable idea, but could you just have it require the bedding to be added, and consumed, in the exact same way as seeds are in a/the trough?

**Comment by rafaelkai** [comment]
It would be better to suggest it here, since it adds more mob compat for Enviromental's Nests:
https://modrinth.com/mod/nest-egg

EDIT: Request it https://github.com/davidjjung/nestegg/issues/5
